benchmarks/vectorized_mosaic.py

```diff
@@ -101,7 +101,9 @@ def _batch_augment(self, inputs):
             minval=0,
             maxval=batch_size,
             dtype=tf.int32,
-            seed=self._random_generator.make_legacy_seed(),
+            seed=self._random_generator.uniform(
```

Comment:
Are we changing the functionality here when we use random uniform to generate seed?